File: src/main/kotlin/eu/alkismavridis/mathasmscript/entities/parser/result/ParseResult.kt
package eu.alkismavridis.mathasmscript.entities.parser.result

import eu.alkismavridis.mathasmscript.entities.logic.FixedMasStatement

class ParseResult(
        val success: Boolean,
        val scriptName: String,
        val packageName: String,
        val imports: Collection<MasImport>,
        val exports: Collection<FixedMasStatement>,
        val comments: Collection<InspectionComment>
)

File: frontend/src/entities/model/ParseResult.ts
import FixedMasStatement from "./FixedMasStatement";
import InspectionComment from "./InspectionComment";
import MasImport from "./MasImport";

export default class ParseResult {
    success: boolean;
    scriptName: string;
    packageName: string;
    imports: MasImport[];
    exports: FixedMasStatement[];
    comments: InspectionComment[];
}
